viewer/vueapp/src/App.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div v-if="compatibleBrowser">
-    <moloch-navbar></moloch-navbar>
+    <moloch-navbar :settings="user.settings">
```

Comment:
Why do we pass it from here instead of getting from store like the other components?

Reply:
Good eye! User is already in the navbar component 👍